Regression/prepare_file_travis.py

```diff
@@ -98,7 +99,17 @@ def select_tests(blocks, match_string_list, do_test):
             blocks = [ block for block in blocks if match_string in block ]
     return blocks
 
-if ci_regular_cartesian:
+if ci_regular_cartesian_2d:
+    test_blocks = select_tests(test_blocks, ['DIM=2'], True)
```

Comment:
There seem to be no test in this one: 4min runtime with `No valid tests defined`
Maybe:
```suggestion
    test_blocks = select_tests(test_blocks, ['DIM=3'], False)
```
?